concent_api/utils/logging.py

```diff
@@ -0,0 +1,71 @@
+from logging                        import getLogger
+
+from golem_messages.message.base    import Message
+
+from utils.helpers                  import get_task_id_from_message
+
+
+logger = getLogger(__name__)
+
+
+def log_message_received(message: Message, client_public_key: str):
+    logger.info('A message has been received in `send/` -- MESSAGE_TYPE: {} -- TASK_ID: {} -- CLIENT PUBLIC KEY: {}'.format(
+        message.TYPE,
+        get_task_id_from_message(message) or '',
+        client_public_key,
+    ))
+    logger.debug('A message has been received in `send/` -- MESSAGE: {}'.format(
+        message,
+    ))
+
+
+def log_message_returned(message: Message, client_public_key: str):
+    logger.info('A message has been returned from `send/` -- MESSAGE_TYPE: {} -- TASK_ID: {} -- CLIENT PUBLIC KEY: {}'.format(
+        message.TYPE,
+        get_task_id_from_message(message) or '',
+        client_public_key,
+    ))
+    logger.debug('A message has been received in `send/` -- MESSAGE: {}'.format(
+        message,
+    ))
+
+
+def log_message_accepted(message: Message, client_public_key: str):
+    logger.info('The message has been accepted for further processing -- MESSAGE_TYPE: {} -- TASK_ID: {} -- CLIENT PUBLIC KEY: {}'.format(
+        message.TYPE,
+        get_task_id_from_message(message) or '',
+        client_public_key,
+    ))
+
+
+def log_message_added_to_queue():
+    logger.info('A new message has been added to queue.')
+
+
+def log_message_delivered():
+    logger.info('A message in queue has been marked as delivered.')
+
+
+def log_deadline_exceeded(message: Message, client_public_key: str, deadline: int):
```

Comment:
Please call it a timeout (i.e. `log_timeout()` here, and `TIMEOUT` in message text). We use `deadline` to refer specifically to the time limits stored inside inside some specific messages, like `TaskToCompute` and calling every timeout a deadline is confusing.

timeout = an event of some time limit or a deadline being exceeded
deadline = the timestamp stored in some specific message types that indicates when their time limit will be exceeded

Reply:
:+1: